libraries/edc-extension/src/main/resources/META-INF/services/org.eclipse.edc.spi.system.ServiceExtension

```diff
@@ -0,0 +1,22 @@
+#################################################################################
+#  Copyright (c) 2024 Robert Bosch Manufacturing Solutions GmbH and others
```

Comment:
Only for my understanding: 
Why we need this file?

Reply:
I think EDC is using Java Service Provider Interfaces for loading extensions. This file defines the names of the extension classes which need to be loaded.

Reply:
thanks for the info